tests/r7rs/r7rs-tests.scm

```diff
@@ -505,22 +512,24 @@
       ((underscore2 (a _) ...) 42)))
   (test 42 (underscore2 (1 2))))
 
-(define-syntax count-to-2
+; TODO(higepon): duplicate pattern variable.
+#;(define-syntax count-to-2
   (syntax-rules ()
     ((_) 0)
     ((_ _) 1)
     ((_ _ _) 2)
     ((_ . _) 'many)))
-(test '(2 0 many)
+#;(test '(2 0 many)
     (list (count-to-2 a b) (count-to-2) (count-to-2 a b c d)))
 
-(define-syntax count-to-2_
+; TODO(higepon): duplicate pattern variable.
+#;(define-syntax count-to-2_
```

Comment:
R6RS `syntax-rules` (in `syntax-case`) cannot handle this case; `_` used as literal. 

https://github.com/okuoku/yuni/blob/master/doc/sibr/SIBR0003-SYNRULE-LIT-UNDERSCORE.md